### PR TITLE
refactor(@angular/ssr): export tokens from the `/tokens` entry point

### LINKED ISSUE #28670: Server code goes to browser bundle on v19
### Command

build

### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### The previous version in which this bug was not present was

_No response_

### Description

Hi. I've noticed some strange code coming to prod build on v19 (+outputMode server + ssr enabled + prerender disabled) during lighthouse test:
 
1) Some kind of server-only error:
Image

2) Postcss?
Image

3) Unvenv with a bunch of polyfills for node's fs?

Image

Correct me if i am wrong, but should this all (50+kb unminified) code come to browser build? Is this hydration-related packages?

### Minimal Reproduction

ng build

### Exception or Error

_No response_

### Your Environment

latest v19

### Anything else relevant?

_No response_

### PR DESCRIPTION
**Pull request by alan-agius4**
This commit relocates the DI tokens to a dedicated entry point. This change ensures that we do not depend on tree-shaking to eliminate server code from client bundles.

Closes #28670

### CODE REVIEW

File: packages/angular/ssr/tokens/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+ts_library(
+    name = "tokens",
+    srcs = glob(
+        [
+            "*.ts",
+            "src/**/*.ts",
+        ],
+    ),
+    module_name = "@angular/ssr/tokens",
```
Comment (alan-agius4) [inline packages/angular/ssr/tokens/BUILD.bazel:13]: Any other ideas for as an entry-point name?

Comment (dgp1130) [reply packages/angular/ssr/tokens/BUILD.bazel:13]: `@angular/ssr/tokens` seems reasonable to me.

**Review by dgp1130 — APPROVED** [review]
Is the goal to support users doing:

```typescript
import { REQUEST } from '@angular/ssr/tokens';

inject(REQUEST, { optional: true });
```

And be able to run that on the client without a risk of retaining server code?

I can see some users having encountering friction from having to import the non-default entry point, but IDE auto-imports should mostly solve that problem, and reducing the risk of server code in the client bundle seems worth the trade-off.

**Comment by alan-agius4** [comment]
> Is the goal to support users doing:
> 
> ```ts
> import { REQUEST } from '@angular/ssr/tokens';
> 
> inject(REQUEST, { optional: true });
> ```
> 
> And be able to run that on the client without a risk of retaining server code?
> 
> I can see some users having encountering friction from having to import the non-default entry point, but IDE auto-imports should mostly solve that problem, and reducing the risk of server code in the client bundle seems worth the trade-off.

Correct.

**Pull request merged**